apt/minimization/minimizer.py

```diff
@@ -258,7 +311,7 @@ def fit(self, X=None, y=None):
         # Return the transformer
         return self
 
-    def transform(self, X):
+    def transform(self, X: Union[np.ndarray, pd.DataFrame]):
```

Comment:
Please update the description of the return value to indicate that it is numpy or pandas according to the input type.